atomic.go

```diff
@@ -304,6 +305,27 @@ func (f *Float64) CAS(old, new float64) bool {
 	return atomic.CompareAndSwapUint64(&f.v, math.Float64bits(old), math.Float64bits(new))
 }
 
+// Duration is an atomic wrapper around time.Duration
+// https://godoc.org/time#Duration
+type Duration struct {
+	v *Int64
```

Comment:
would it make more sense to reimplement the underlying int64 bit so we don't have a double-pointer situation?

Reply:
don't need to reimplement, you can just make this `v Int64`

Reply:
Use `Int64` instead of a pointer